src/pages/groups/models/mockData.ts

```diff
@@ -196,12 +208,118 @@ export function mockGroup(): ResourceResponseInclude<Group, Contact> {
       categories: relatedCollection(`/${code}/categories`, 11),
       offers: relatedCollection(`/${code}/offers`, 222),
       needs: relatedCollection(`/${code}/needs`, 12),
-      posts: relatedCollection(`/${code}/posts`, 34),
+      posts: relatedCollection(`/${code}/posts`, 34)
     }
-  }
+  };
 
   return {
     data: group,
     included: contacts
+  };
+}
+
+function mockCategorySummary(index: number): Category {
+  const id = uid();
+  return {
+    id: id,
+    type: "categories",
+    attributes: {
+      code: lorem.generateWords(1),
+      name: lorem.generateWords(Math.round(Math.random() * 2) + 1),
+      cpa: ["10", "11", "56"],
+      description: lorem.generateParagraphs(Math.round(Math.random() * 4) + 1),
+      icon: testImages[index % testImages.length],
+      access: "public",
+      created: new Date().toJSON(),
+      updated: new Date().toJSON()
+    },
+    relationships: {
+      group: {
+        links: {
+          related: "https://komunitin.org/EITE"
+        }
+      },
+      needs: {
+        links: {
+          related: "https://komunitin.org/EITE/needs?filter[category]=food"
+        },
+        meta: {
+          count: 3
+        }
+      },
+      offers: {
+        links: {
+          related: "https://komunitin.org/EITE/offers?filter[category]=food"
+        },
+        meta: {
+          count: 32
+        }
+      }
+    },
+    links: {
+      self: BASE_URL + "/categories/" + id
+    }
+  };
+}
+/**
+ * Mock result for GET /{groupCode}/categories
+ *
+ * https://app.swaggerhub.com/apis/estevebadia/komunitin-api/0.0.1#/Groups/get_groups
+ */
+export function mockCategoryList(): CollectionResponse<CategorySummary> {
+  const list = [] as CategorySummary[];
+
+  for (let index = 1; index < NUM_CATEGORIES; index++) {
+    list.push(mockCategorySummary(index));
   }
-}
\ No newline at end of file
+
+  return {
+    data: list,
+    links: {
+      self: BASE_URL + "/categories",
+      first: BASE_URL + "/categories",
+      prev: null,
+      next: null
+    },
+    meta: {
+      count: Math.round(Math.random() * 500)
```

Comment:
```suggestion
      count: NUM_CATEGORIES
```

Reply:
Estoy utilizando NUM_CATEGORIES para el número de categories que queremos pero el siguiente paso ya no hará falta o lo dejo como por defecto en caso de no recibir el parámetro.

Reply:
`count` representa el numero total de `categories` que existen en el sistema para un grupo dado. No es esto NUM_CATEGORIES?

Reply:
No entiendo lo del siguiente paso

Reply:
NUM_CATEGORIES se utiliza en el for que genera las categorías, no es el total.
En estos momentos está sustituyendo el perPag que en el siguiente paso llegara por parámetro.

Reply:
Ten en cuenta, para la paginación, ordenación y demás, de seguir las recomendaciones jsonapi.org. Yo creo que lo dejaría de momento y lo tratasría en una issue a parte

Reply:
Si, estoy siguiendo las directivas de jsonapi.org.
Necesitaba diferenciar entre offers y needs en la llamada.
De momento hago lo justo para poder ir avanzando en tener las páginas.
La idea ya es lo que dices.